src/services/rfqm_service.ts

```diff
@@ -186,8 +195,13 @@ export class RfqmService {
         const marketOperation = isSelling ? MarketOperation.Sell : MarketOperation.Buy;
         const assetFillAmount = isSelling ? sellAmount! : buyAmount!;
 
-        // Prepare gas estimate
+        // Prepare gas estimate and fee
         const gas: BigNumber = await this._protocolFeeUtils.getGasPriceEstimationOrThrowAsync();
+        const fee: RfqmFee = {
+            amount: ZERO,
```

Comment:
Just to refresh my memory, we we going to compute the fee in a second PR right?

Reply:
correct! Wanted to make sure we were able to focus on that one separately